### PR TITLE
feat(utils): add EmbedsBuilder.addFields()

### PR DESCRIPTION
**Pull request by mezotv**
This implements a simple addFields function, similar to setFields, but instead of overwriting the previous fields it just adds them to the array. I also added a test for the addField function and fixed a typo in the tsdoc from earlier.

### CODE REVIEW

**Comment by mezotv** [comment]
Ill try to fix the tests later moving this to a draft until that is figured out!

**Review by itohatweb — CHANGES_REQUESTED** [review]
Would it be possible to move the renaming to `EmbedsBuilder` to a separate PR? So we can avoid needing to bump a major version a bit longer.

**Comment by mezotv** [comment]
> Would it be possible to move the renaming to `EmbedsBuilder` to a separate PR? So we can avoid needing to bump a major version a bit longer.

Its not actually a rename just changes the tsdoc. The class was always named like this. But yea if needed I can make a seperate pr for it!

File: packages/utils/src/builders/embeds.ts [hunk 1]
```diff
@@ -43,6 +43,18 @@ export class EmbedsBuilder extends Array<DiscordEmbed> {
...
   }
 
+  /**
+   * Adds multiple new fields to the embed fields array
+   *
+   * @param {DiscordEmbedField[]} fields
+   * @returns {EmbedsBuilder}
+   */
+  addFields(fields: DiscordEmbedField[]): this {
+    this.#currentEmbed.fields.push(fields)
```
Comment (Fleny113) [inline packages/utils/src/builders/embeds.ts:53]:
```suggestion
    this.#currentEmbed.fields.push(...fields)
```

File: packages/utils/src/builders/embeds.ts [hunk 2]
```diff
@@ -37,12 +37,28 @@ export class EmbedsBuilder extends Array<DiscordEmbed> {
...
+      ...(inline !== undefined && { inline }),
     })
 
     return this
   }
 
+  /**
+   * Adds multiple new fields to the embed fields array
+   *
+   * @param {DiscordEmbedField[]} fields
```
Comment (Fleny113) [inline packages/utils/src/builders/embeds.ts:49]:
```suggestion
   * @param {DiscordEmbedField[]} fields - The fields to add
```

**Review by Fleny113 — DISMISSED** [review]

**Review by Fleny113 — DISMISSED** [review]

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**